lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -83,9 +74,23 @@ class FirstMeaningfulPaint extends Audit {
     });
   }
 
-  static calculateScore(data) {
-    // First meaningful paint is the last timestamp observed from the candidates
-    const firstMeaningfulPaint = (data.fMP.ts - data.navStart.ts) / 1000;
+  static calculateScore(evts) {
+    const firstMeaningfulPaint = (evts.firstMeaningfulPaint.ts - evts.navigationStart.ts) / 1000;
+    const firstContentfulPaint = (evts.firstContentfulPaint.ts - evts.navigationStart.ts) / 1000;
```

Comment:
Just checking...I've seen multiple FCP events in a trace. https://bugs.chromium.org/p/chromium/issues/detail?id=673922. Will this need to be more robust to handle those cases?